css/roboto-fontface.scss

```diff
@@ -28,14 +29,18 @@ $roboto-font-path: '../fonts' !default;
     }
 }
 
-@mixin roboto-font-pair($type, $weight) {
+@mixin roboto-font-pair($type, $weight, $with-italic: true) {
     @include roboto-font($type, $weight);
-    @include roboto-font(#{$type}Italic, $weight, italic);
+    @if $with-italic {
+        @include roboto-font(#{$type}Italic, $weight, italic);
+    }
 }
 
-@include roboto-font-pair('Thin', 100);
-@include roboto-font-pair('Light', 300);
-@include roboto-font-pair('Regular', 400);
-@include roboto-font-pair('Medium', 500);
-@include roboto-font-pair('Bold', 700);
-@include roboto-font-pair('Black', 900);
+@if $include-styles {
```

Comment:
Ah, now I see what you are aiming for:

In less there is a feature called reference import. This way you can import the definitions (like mixins and so) so that you can use the mixins, but you do not get any additional css implicitly.

Since I do not use scss myself, I looked up [here](http://stackoverflow.com/questions/22851662/what-is-the-equivalent-of-lesss-import-reference-style-in-sass), that there is no similar feature in scss. But I might find it better, to not have your proposed `$include-styles` toggle variable, but instead split the scss file up like so:

```scss
// _mixins.scss
$roboto-font-path: '../fonts' !default;

@mixin roboto-font($type, $weight, $style: normal) {
    @font-face {
        font-family: 'Roboto';
        src: url('#{$roboto-font-path}/Roboto-#{$type}.eot');
        src: local('Roboto #{$type}'),
             local('Roboto-#{$type}'),
             url('#{$roboto-font-path}/Roboto-#{$type}.eot?#iefix') format('embedded-opentype'),
             url('#{$roboto-font-path}/Roboto-#{$type}.woff2') format('woff2'),
             url('#{$roboto-font-path}/Roboto-#{$type}.woff') format('woff'),
             url('#{$roboto-font-path}/Roboto-#{$type}.ttf') format('truetype'),
             url('#{$roboto-font-path}/Roboto-#{$type}.svg#Roboto') format('svg');
        font-weight: $weight;
        font-style: $style;
    }

    @font-face {
        font-family: 'Roboto-#{$type}';
        src: url('#{$roboto-font-path}/Roboto-#{$type}.eot');
        src: local('Roboto #{$type}'),
             local('Roboto-#{$type}'),
             url('#{$roboto-font-path}/Roboto-#{$type}.eot?#iefix') format('embedded-opentype'),
             url('#{$roboto-font-path}/Roboto-#{$type}.woff2') format('woff2'),
             url('#{$roboto-font-path}/Roboto-#{$type}.woff') format('woff'),
             url('#{$roboto-font-path}/Roboto-#{$type}.ttf') format('truetype'),
             url('#{$roboto-font-path}/Roboto-#{$type}.svg#Roboto') format('svg');
    }
}

@mixin roboto-font-pair($type, $weight) {
    @include roboto-font($type, $weight);
    @include roboto-font(#{$type}Italic, $weight, italic);
}
```

```scss
// roboto-fontface.scss
@import "mixins";

@include roboto-font-pair('Thin', 100);
@include roboto-font-pair('Light', 300);
@include roboto-font-pair('Regular', 400);
@include roboto-font-pair('Medium', 500);
@include roboto-font-pair('Bold', 700);
@include roboto-font-pair('Black', 900);
```

This way we stay clean, but if you only want a certain subset of font definitions, you can just import the `_mixins.scss` and just pick the `@include roboto-font-pair` instructions you need. The italic switch seems ok.

What do you think?